qt_ui/widgets/QTopPanel.py

```diff
@@ -127,15 +130,22 @@ def setGame(self, game: Optional[Game]):
         self.budgetBox.setGame(game)
         self.factionsInfos.setGame(game)
 
-        self.passTurnButton.setEnabled(True)
-        if game and game.turn > 0:
-            self.passTurnButton.setText("Pass Turn")
+        state = game.check_win_loss()
+        enabled = state == TurnState.CONTINUE
+        self.passTurnButton.setEnabled(enabled)
+        self.proceedButton.setEnabled(enabled)
+        self.simSpeedControls.setEnabled(enabled)
+        self.air_wing.setEnabled(enabled)
+        self.transfers.setEnabled(enabled)
 
-        if game and game.turn == 0:
```

Comment:
`if game` <=> `if game is not None` for which we have already tested at the beginning of the method, no need to check it again...